python/semantic_kernel/connectors/memory/astradb/astradb_memory_store.py

```diff
@@ -0,0 +1,300 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import logging
+from typing import List, Optional, Tuple
+
+from numpy import ndarray
+
+from semantic_kernel.connectors.memory.astradb.astra_client import AstraClient
+from semantic_kernel.connectors.memory.astradb.utils import (
+    build_payload,
+    parse_payload,
+)
+from semantic_kernel.memory.memory_record import MemoryRecord
+from semantic_kernel.memory.memory_store_base import MemoryStoreBase
+
+import aiohttp
+
+MAX_DIMENSIONALITY = 20000
+MAX_UPSERT_BATCH_SIZE = 100
+MAX_QUERY_WITHOUT_METADATA_BATCH_SIZE = 10000
+MAX_QUERY_WITH_METADATA_BATCH_SIZE = 1000
+MAX_FETCH_BATCH_SIZE = 1000
+MAX_DELETE_BATCH_SIZE = 1000
+
+
+class AstraDBMemoryStore(MemoryStoreBase):
+    """A memory store that uses Astra database as the backend."""
+
+    logger: logging.Logger = logging.getLogger(__name__)
+
+    def __init__(
+        self,
+        astra_application_token: str,
+        astra_id: str,
+        astra_region: str,
+        keyspace_name: str,
+        embedding_dim: int,
+        similarity: str,
+        logger: Optional[logging.Logger] = None,
```

Comment:
we no longer use this setup for loggers, please remove and use regular "pythonic" logging (`import logging, logger = logging.get_logger(__name__)`)

Reply:
Done